### PR TITLE
Integration tests for us043

### PR DESCRIPTION
**Pull request by tej99**
Feature is in master branch
As an: 
internal User
I need: 
to be able to 'find' a Reporting Unit (RU Ref for Business Surveys)
So that: 
I can easily locate the RU to deal with any issues

### CODE REVIEW

**Comment by tej99** [comment]
[US043: INT Find an RU [Large]](https://trello.com/c/WyRWFKXq/38-us043-int-find-an-ru-large)

File: acceptance_tests/features/find_reporting_unit.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Feature: Find Reporting Unit
+   As an internal user
+   I need to be able to find a Reporting Unit
+   So that I can easily locate the RU to deal with issues
```
Comment (andrewmil) [inline acceptance_tests/features/find_reporting_unit.feature]: I think we should be consistent with saying 'RU' or 'Reporting unit' In general I'm leaning towards the latter

File: acceptance_tests/features/pages/reporting_units.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from acceptance_tests import browser
```
Comment (andrewmil) [inline acceptance_tests/features/pages/reporting_units.py]: I think this file should be called `reporting_units_search` if we're going to have a file in this folder per page

File: acceptance_tests/features/steps/find_reporting_unit.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    reporting_units.search_reporting_unit('49900000005')
+
+
+@then('the Reporting Unit is displayed back to the user')
+def internal_user_presented_with_correct_ru(_):
+    rus = reporting_units.get_reporting_units()
+    assert '49900000005' in rus and 'OFFICE FOR NATIONAL STATISTICS' in rus
+
+
+@when('the internal user enters a RU Name')
```
Comment (andrewmil) [inline acceptance_tests/features/steps/find_reporting_unit.py]: Maybe have this when above the `then` statements?

File: acceptance_tests/features/pages/reporting_units.py (hunk 2 above)
Comment (tej99) [reply acceptance_tests/features/pages/reporting_units.py]: It's the /reporting-units page which is why I named it reporting_units.py the reporting unit details page should be called reporting_units_details.py to match the current pattern we have with collection_exercise.py and collection_exercise_details.py.

**Pull request merged**